packages/components/Layout/Logo.js

```diff
@@ -20,10 +21,15 @@ export const Logo = ({
   iconName = 'logo-caesar-4xxi',
   width = DEFAULT_LOGO_WIDTH,
   height = DEFAULT_LOGO_HEIGHT,
-}) => (
-  <Link passHref href={href}>
-    <LogoLink>
+}) =>
+  APP_TYPE === 'secure' ? (
```

Comment:
Please use the app type env https://github.com/caesar-team/caesar-web-client/blob/master/packages/common/constants.js#L25

Reply:
done